README.md

```diff
@@ -373,6 +372,21 @@ todo: add response example
 
 ```
 
+#### user.positions.getList({ txId })
+
+This method retrieves a single transaction.
```

Comment:
2 things: why is it "transaction" here, and why "single"?

Reply:
Thx, fixed